docs/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+---
+Lighthouse is an open-source project to build and manage data lakes. Lighthouse is built on top of Scala and Spark. Using lighthouse, you can define the data sources available in your data lake and easily build data pipelines based on those data sources.  
```

Comment:
Maybe add the benefits of using lighthouse here as well. As it is not fully clear why a user would want to use lighthouse